client/pages/client/create.js

```diff
@@ -20,11 +20,16 @@ export default function NewClient({ url }) {
             email: clientEmail,
             address: clientAddress,
         }
-        const resp = await fetch(`${url}/client`, {
+        await fetch(`${url}/client`, {
             method: 'POST',
             mode: 'cors',
             body: JSON.stringify(newClient)
         })
+        .then(async (res) => {
```

Comment:
This sort of logic is in a couple places and even though it may work, it isn't quite right. It's mixing `Promise.then()` with `async/await`. This should probably be switched to pure async/await since the `submitClient` function is already `async` and `await` syntax is much cleaner